app/src/main/java/de/xikolo/network/jobs/base/NetworkJob.kt

```diff
@@ -33,6 +34,12 @@ abstract class NetworkJob(private val networkState: NetworkStateLiveData, privat
 
     fun error() = networkState.error(userRequest)
 
+    fun deprecated(deprecationDate: Date) = networkState.deprecated(deprecationDate, userRequest)
+
+    fun apiVersionExpired() = networkState.apiVersionExpired(userRequest)
+
+    fun maintenance() = networkState.maintenance(userRequest)
```

Comment:
Since these methods are only used by `CheckHealthJob` they should be moved there.

Reply:
That's right.